drivers/KS-Delta/drive-ks.cpp

```diff
@@ -0,0 +1,1413 @@
+/*BHEADER**********************************************************************
```

Comment:
@cadaverous-lives I would add a brief mention of the new drivers in the 
drivers/README.md 
file